src/blister.h

```diff
@@ -48,6 +48,7 @@ EXPORT_INLINE Int SIZE_PLEN_BLIST(Int plen)
     return sizeof(Obj) + (plen + BIPEB - 1) / BIPEB * sizeof(UInt);
 }
 
+
```

Comment:
Why did you add this line? This is not consistently done in the GAP kernel. Do we want to have two empty lines before we habe a `/***[...]` line?

Reply:
I generally aim for two empty lines, but this is not codified anywhere (and I am not keen on trying to codify it explicitly or officially either, as I am afraid that will just lead down a road of endless bikeshedding). In this particular case, the functions before and after this all used two empty lines (later on in the file it is again inconsistent). I guess I inserted the empty line (two years ago...) out of reflex. I can drop it, but I think it doesn't hurt either.

Reply:
My comment was more of a question than an intend to get you to change it back. I'm fine with it, I just wanted to know what the general philosophy was.